.github/workflows/cron.yml

```diff
@@ -25,8 +25,9 @@ jobs:
           ORG_LOGIN: ${{ secrets.ORG_LOGIN }}
         with:
           script: |
-            const script = require('./main.js')
-            console.log(script({github, context}))
+            const { default: manageIPs } = await import('${{ github.workspace }}/main.js')
```

Comment:
Out of curiosity, how come we changed how we call the main method? Did something break?

Reply:
Since I set it as a module the workflow was breaking at `require` used the ESM import example provided by (github-scripts)[https://github.com/actions/github-script#use-esm-import]. This can be ironed out but it resolves the issues when running the workflow